src/myq-garagedoor.ts

```diff
@@ -224,12 +205,8 @@ export class myQGarageDoor extends myQAccessory {
       this.log.error("%s: Unable to %s door while currently attempting to complete %s. myQ must complete it's existing action before attempting a new one.",
         accessory.displayName, actionAttempt, actionExisting);
 
-      if(callback) {
-        callback(new Error("Unable to accept a new set event while another is completing."));
-      }
-
+      //new Error("Unable to accept a new set event while another is completing."));
```

Comment:
not sure how you want to handle this?

Reply:
Same as below...log the error, call it a day.

Reply:
Ditto...it already returns false if it errors out.